manifest.json

```diff
@@ -0,0 +1,65 @@
+{
+    "name": "Trading App",
+    "short_name": "Trading App",
+    "start_url": "/index.html",
+    "display": "standalone",
+    "background_color": "#000",
+    "theme_color": "#000",
+    "scope": ".",
+    "orientation": "portrait-primary",
+    "icons": [
+        {
+            "src": "src/assets/icons/icon-48x48.png",
+            "sizes": "48x48",
+            "type": "image/png",
+            "purpose": "maskable"
+        },
+        {
+            "src": "src/assets/icons/icon-72x72.png",
+            "sizes": "72x72",
+            "type": "image/png",
+            "purpose": "maskable"
+        },
+        {
+            "src": "src/assets/icons/icon-96x96.png",
+            "sizes": "96x96",
+            "type": "image/png",
+            "purpose": "maskable"
+        },
+        {
+            "src": "src/assets/icons/icon-128x128.png",
+            "sizes": "128x128",
+            "type": "image/png",
+            "purpose": "maskable"
+        },
+        {
+            "src": "src/assets/icons/icon-144x144.png",
+            "sizes": "144x144",
+            "type": "image/png"
```

Comment:
You can provide `purpose:any` for icons that u wish to use as non-maskable. Else u'll get lighthouse errors :)